### PR TITLE
Job description

### PR DESCRIPTION
**Pull request by S1ss3l**
## About the PR 
Added job description in english ! I already made a PR (#12275) for the first commit but I don't know how to remove it from my branch. I guess it should be fine since the changes are identical.

:cl:
- add: You should now see a description when hovering over job title.

### CODE REVIEW

**Review by keronshb — COMMENTED** [review]
Left a review, it's all spelling and some formatting.

File: Resources/Locale/en-US/job/job-description.ftl [hunk 1]
```diff
@@ -0,0 +1,42 @@
+job-description-assistant = An engineer in training, help the engineer fix the station and try to learn atmos !
```
[the remark this answers is not in the document]
Comment (S1ss3l) [reply Resources/Locale/en-US/job/job-description.ftl:1]: The localization for the name use the same scheme, should I change it too ? (it's job-name-assistant)

**Comment by S1ss3l** [comment]
I'm happy that the only thing to change are typo (english not being my main language yada yada yada). I'll wrap this all up in a commit soon, thanks for the feedback!

File: Resources/Locale/en-US/job/job-description.ftl (hunk 1 above)
[the remark this answers is not in the document]
Comment (keronshb) [reply Resources/Locale/en-US/job/job-description.ftl:1]: Oh yeah I see that now. That should be renamed to job-name-technical-assistant too, considering assistant.yml is passenger.

**Comment by keronshb** [comment]
> I'm happy that the only thing to change are typo (english not being my main language yada yada yada). I'll wrap this all up in a commit soon, thanks for the feedback!

Yep no problem, glad to help!

File: Resources/Locale/en-US/job/job-description.ftl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+job-description-passenger = Enjoy your stay! Get in trouble for breaking into rooms. Save the station when least expected.
+job-description-psychologist = Provide emotional support to traumatized crew. Have the statu of a doctor without doing any of the work.
+job-description-qm = Organize all of cargo into a logistical force to be reckoned with. Yell at HoP for giving the clown cargo access.
+job-description-rd = Make sure the scientists are taking their jobs seriously and improving the station as much as they can.
+job-description-reporter = Pester scientists for a wireless camera, make a news channel, spend all your time documenting drunken rants in the bar.
+job-description-salvagespec = Equip a hardsuit and salvage wrecked station parts from previous stations. Fight space carp and space bear.
+job-description-scientist = Build machine, unlock technology, make your own drug lab, mass produce grenade, pester cargo for more materials.
+job-description-security = Enforce space law, cuff troublemaker and bring them to brigg, be fair and try not to get hated by half the station.
```
Comment (mirrorcult) [inline Resources/Locale/en-US/job/job-description.ftl:39]:
```suggestion
job-description-security = Enforce the law, cuff troublemakers and bring them to the brig, be fair and try not to get hated by half the station.
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+job-description-cargotech = Manage order, make deliveries, supply material to scientist, seeds to botanist and laser gun to security.
+job-description-ce = Order the engineers around to make sure that all issues in the station are fixed without any delays.
+job-description-centcomoff = CentCom official, oversee the clownfest happening in the 14th space station.
+job-description-chaplain = Preach the word of the holy toolbox. Get godly power from your faith. Render medbay obsolete
+job-description-chef = Keep the station fed, protect your cleaver at all costs, explore "alternative" meat sources.
+job-description-chemist = Use the chemistry machines to make medicine, get bored and start mass producing meth.
+job-description-clown = Make the station laugh. Tell jokes. Perform elaborate pranks. Being a clown is not an excuse to be an asshole.
+job-description-cmo = Manage the resources and personnel of the medical department to keep the crew in top shape.
+job-description-detective = Investigate crime. Free innocent falsly condamned and incriminate real criminal (and vice-versa).
```
Comment (mirrorcult) [inline Resources/Locale/en-US/job/job-description.ftl:16]:
```suggestion
job-description-detective = Investigate crime. Free innocent falsly condamned and incriminate real criminal (and vice-versa).
```
this just needs to be completely rewritten, it doesn't make any sense

File: Resources/Locale/en-US/job/job-description.ftl [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+job-description-qm = Organize all of cargo into a logistical force to be reckoned with. Yell at HoP for giving the clown cargo access.
+job-description-rd = Make sure the scientists are taking their jobs seriously and improving the station as much as they can.
+job-description-reporter = Pester scientists for a wireless camera, make a news channel, spend all your time documenting drunken rants in the bar.
+job-description-salvagespec = Equip a hardsuit and salvage wrecked station parts from previous stations. Fight space carp and space bear.
+job-description-scientist = Build machine, unlock technology, make your own drug lab, mass produce grenade, pester cargo for more materials.
+job-description-security = Enforce space law, cuff troublemaker and bring them to brigg, be fair and try not to get hated by half the station.
+job-description-serviceworker = Be a sightly more useful passenger. Observe the other civilian jobs before trying them out.
+job-description-warden = Manage the armory and prisoneer. Prevent the prisoners from being forgotten in their cells.
```
Comment (mirrorcult) [inline Resources/Locale/en-US/job/job-description.ftl:41]:
```suggestion
job-description-warden = Manage the armory and brig. Prevent the prisoners from being forgotten in their cells.
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]
these descriptions still have a lot of typos and I just don't like them overall, you should really do a second pass/get someone else to do these

**Pull request closed without merging**